index.html

```diff
@@ -49,6 +50,182 @@ <h2>LET’S CONNECT</h2>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="tags">
+                <ul>
+                  <li class="tags1"><h3>HTML</h3></li>
+                  <li class="tags2"><h3>CSS</h3></li>
+                  <li class="tags3"><h3>Javascript</h3></li>
+                </ul>
+              </div>
+              <div class="see-more-detail">
+                <div class="button">
+                  <a href="#">
+                    See Project
+                  </a>
+                </div>
+              </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    
+    <section class="about">
+      <div class="about-content">
+        <div class="about-content-top">
+          <h3>About <br/> Myself</h3>
+          <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <div class="social-about">
+            <div class="social-text-about">
+              <h2>LET'S CONNECT</h2>
+            </div>
+            <div class="social-icon-about">
+              <ul>
+                <li><a href="#"><img src="assets/images/mobile/twitter.svg" class="twitter-icon-about" alt="image"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/github.svg" class="github-icon-about" alt="image"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/Linkedin-icon.svg" class="linkedin-icon-about" alt="image"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/m.svg" class="about-m-icon" alt="image-about"></a></li>
+                <li><a href="#"><img src="assets/images/mobile/finger.svg" class="finger-icon-about" alt="image"></a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="resume">
+            <a href="#">
+              Get my resume
+            </a>
+          </div>
+        </div>
+        <div class="skills">
+          <ul>
+            <li class="languages">
+              <div class="lang-frame">
+                <h3>Langguages</h3>
+                <img src="assets/images/mobile/dropdown-Icon.png" alt="image">
+              </div>
+              <ul class="lang-list">
+                <li class="lang-js"><h3>Javascript</h3></li>
+                <li class="lang-htm"><h3>HTML</h3></li>
+                <li class="lang-cs"><h3>CSS</h3></li>
+              </ul>
+            </li>
+            <li class="framework">
+              <h3>Frameworks</h3>
+              <img src="assets/images/mobile/Icon-Right-Larger.svg" alt="">
+            </li>
+            <li class="skill">
+              <h3>Skills</h3>
+              <img src="assets/images/mobile/Icon-Right-Larger.svg" alt="">
+            </li>
+          </ul>
+        </div>
+      </div>
```

Comment:
Please improve the design of the section in the image
- [ ] **Languages**, **Frameworks** and **Skills** should be bolder as defined in the template
- [ ] **Frameworks** and **Skills** should some kind of border-bottom as in the template
- [ ] The text **HTML** should be somewhat vertically aligned in the middle just like **Javascript** and **CSS**
- [ ] The images should have some top and left margins as in the template.

![emma-langs](https://user-images.githubusercontent.com/36542986/134600120-1029599c-7070-46f0-89ee-65c052c07055.jpg)